Apply correct and additional coding style rules

AJTimmerman opened the pull request:
DVO-59

Commit:
Apply correct and additional coding style rules

DVO-59

Commit:
Add phpdoc alignment

DVo-59

cvanzanten-dij commented:
![Screenshot 2024-07-29 at 06 48 32](https://github.com/user-attachments/assets/029a0481-4cdb-4351-b5f5-11ef64e3be5d)
Ik wil graag expliciete null markering behouden.

![Screenshot 2024-07-29 at 06 50 02](https://github.com/user-attachments/assets/17354349-c23a-463b-9373-4484fe432f05)
Spaties om de string concat operator wil ik ook graag behouden.

AJTimmerman commented:
@cvanzanten-dij hmm, ik ga kijken waar dit vandaan komt. Beide opties had ik niet verwacht, dus die regels loop ik direct na. Thanks voor het zorgvuldig checken!

Commit:
Remove deprecated rule

DVO-59

Commit:
Format single line empty body to multiline

The laravel preset has this on true, which conflicts with the coding standards

DVO-59

AJTimmerman commented:
Deze changes werken niet als er in een project laravel/pint versie 1.13 wordt gebruikt. 1.14 niet getest, maar 1.15+ werkt wel. Het is dus spicy dat dit niet getagged is, aangezien builds gaan falen als we dit doorvoeren. Scheelt dat de pint update niet zo spannend is, maar kan dus wel voor onverwachte problemen zorgen

Commit:
Add rule to prevent removal of parentheses after the new keyword

DVO-59

Commit:
Don’t enforce yoda style

DVO-59

Commit:
Overwrite Laravel’s preset of phpdoc inheritdoc types

Laravel uses the ‘inline’ rule, which formats @inheritdoc to {@inheritdoc}, which isn’t in our code style

DVO-59

AJTimmerman commented:
Werkzaamheden zijn nu afgerond. Heb wat extra rules moeten toevoegen ten opzichte van de coding-standards aangezien we hier de Laravel preset gebruiken, wat niet overeen komt met onze code styling. Dat is hiermee wel gefixt

cvanzanten-dij reviewed (approved):
Getest en werkt naar verwachting

AJTimmerman commented:
Ook nog even bij Maarten gecheckt, alle vlaggen zijn groen, dus ik merge de PR
